install/preview/entrypoint.sh

```diff
@@ -5,6 +5,10 @@
 
 set -ex
 
+if [ "$1" != "logging" ]; then
```

Comment:
We're leaking a small amount of logs with the `set -x` on line 6, can we move that `set -x` as an else clause to this conditional?

```
❯ docker run --privileged --name gitpod --rm -it -v /tmp/gitpod:/var/gitpod eu.gcr.io/gitpod-core-dev/build/preview-install:tar-preview-output.5
+ '['  '!=' logging ]
+ /entrypoint.sh logging
+ /prettylog
# [pretty logs follow]
```